Makefile

```diff
@@ -241,6 +241,9 @@ test-debugger: all
 test-inspector: all
 	$(PYTHON) tools/test.py inspector
 
+test-node-inspect: $(NODE_EXE)
```

Comment:
Yep, didn't have a Windows machine set up to test that part, that's why I only copied `tools/test-npm.sh` in the first push. Should be able to work on the Windows part next week.